.github/workflows/smoketest.yml

```diff
@@ -346,6 +346,8 @@ jobs:
         # force regression suite to timeout after 8 minutes, so it captures ZAP report of any inflight hung queries
         timeoutcmd $((8 * 60)) \
           ./ecl-test setup --preAbort 'smoketest-preabort.sh' --pq 2 --generateStackTrace
+        grep Failure: ${{ github.workspace }}/regress/log/setup_*.log
+        if [[ "$(grep -ohP '(?<=^Failure: )[0-9]+$' ${{ github.workspace }}/regress/log/setup_*.log | awk '{s+=$1} END {print s}')" -gt 0 ]]; then exit 1; fi
```

Comment:
I think if would be simpler if use '-c' parameter with grep:
```
if [[ "$(grep -ohPc '(?<=^Failure: )[0-9]+$' ${{ github.workspace }}/regress/log/setup_*.log )" -gt 0 ]]; then exit 1; fi
```

Reply:
That's not equivalent.
The current version extracts the Failure counts - then pipes them through awk, to create a single total.
e.g. if setup_thor...log contains:
```
Failure: 3
```
and setup_roxie...log contains:
```
Failure: 4
```

This will produce a result of 7, meaning the test will resolve to [[ 7 -gt 0 ]]; ...

grep -c produces a count per file (of matches), meaning you'll get an output of:
```
1
1
```
if [[ 1 1 -gt 0 ]]; ...

and will cause a syntax error.

Reply:
True.